frontend/packages/dev-console/integration-tests/features/getting-started/sample-card_Add_page.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Create Sample Application
+    As a user I want to create the Sample Application from Add options
+
+Background:
+    And user is in dev perspecitve
```

Comment:
```suggestion
    And user is at Developer perspecitve
```

We have used this thing almost everywhere. It's better to use the same statement

Reply:
`in` works too